deploy/kiali/kiali_cr.yaml

```diff
@@ -580,6 +580,8 @@ spec:
 # url: External URL that will be used to generate links to Jaeger. It must be accessible to clients external to
 #      the cluster (e.g: browser) in order to generate valid links.
 #      If tracing service is deployed in a QUERY_BASE_PATH set this in the url like https://<hostname>/<QUERY_BASE_PATH> . EX: https://tracing-service:8080/jaeger
+# use_grpc: Set "true" to enable GRPC connection between Kiali and Jaeger, in order to speed up the queries. In some setups you might not be able to use
+#      GRPC (e.g. if Jaeger is behind some reverse proxy that doesn't support it).
```

Comment:
```suggestion
#      GRPC (e.g. if Jaeger is behind some reverse proxy that doesn't support it).
#      If not specified, it will be true if deployed within a Maistra+OpenShift environment, false otherwise.
```

Reply:
"If not specified, it will be true if deployed within a Maistra+OpenShift environment, false otherwise."
We need to set use_grpc to false for Service mesh.

Reply:
right, it's the opposite. I'll commit mazz' suggestions then update